source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ContainerRenderer.java

```diff
@@ -308,6 +309,25 @@ public static void setSelectAction(RenderedAdaptiveCard renderedCard, BaseAction
             view.setOnClickListener(new BaseActionElementRenderer.SelectActionOnClickListener(renderedCard, selectAction, cardActionHandler));
 
             applyTitleAndTooltip(selectAction, view);
+
+            if (view instanceof ViewGroup)
+            {
+                ViewGroup group = (ViewGroup) view;
+                if (group.getChildCount() == 1)
+                {
+                    View childView = group.getChildAt(0);
+                    if (childView.isFocusable())
+                    {
+                        childView.setFocusable(false);
+
+                        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P)
```

Comment:
I suggest adding a comment as it is not clear how the build version is related here.

Reply:
Just added a comment explaining this!